coretex/entities/annotation/image/coretex_format.py

```diff
@@ -156,10 +156,10 @@ def centroid(self) -> Tuple[int, int]:
 
         flattenedSegmentations = [element for sublist in self.segmentations for element in sublist]
 
-        listCX = [value for index, value in enumerate(flattenedSegmentations) if index % 2 == 0]
+        listCX = [value for index, value in enumerate(flattenedSegmentations) if index % 2 == 0 and index < 8]
```

Comment:
There can be N number of points in the `flattenedSegmentations` array so an `index` of `8` should not be used as a condition, the last point should be discarded from `flattenedSegmentations` array.